src/Reflex/Dom/SemanticUI/Button.hs

```diff
@@ -124,14 +137,16 @@ uiButtonAttrs UiButton{..} = T.unwords $ catMaybes
 -- provided by other functions such as 'uiButtonAnimated'.
 uiButton
     :: MonadWidget t m
-    => Dynamic t UiButton
+    => UiButtonType
```

Comment:
I'm cool with that. I could add new functions like `uiButtonWith` and have `uiButton` et. al. default to the most common use-case (in Reflex apps that is).